payments-core/src/main/java/com/stripe/android/paymentsheet/injection/PaymentSheetViewModelBindingModule.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.paymentsheet.injection
+
+import android.app.Application
+import android.content.Context
+import dagger.Binds
+import dagger.Module
+
+@Module
+internal interface PaymentSheetViewModelBindingModule {
+    @Binds
+    fun bindsApplicationForContext(application: Application): Context
```

Comment:
@Binds and @Provides can't [live in the same module](https://dagger.dev/dev-guide/faq.html#binds) unless provides are all static, create a separate one for PaymentSheetViewModel

Reply:
Any downside to just making all `@Provides` methods static? I believe the end result is the same, right?

Reply:
Not really, the syntax for adding `abstract fun` and `@JvmStatic` is a bit messy on kotlin, but did some research they do [added support](https://github.com/google/dagger/releases/tag/dagger-2.26) for the mixed approach with `companion object`. updated